FabNeso.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This source file is part of the FabSim software toolkit, which is
+# distributed under the BSD 3-Clause license.
+# Please refer to LICENSE for detailed information regarding the licensing.
+#
+# This file contains FabSim definitions specific to FabNeso
+#
+# authors: Duncan Leggat
+
+try:
+    from fabsim.base.fab import *
+except ImportError:
+    from base.fab import *
+
+add_local_paths("FabNeso")
+
+@task
+def neso(config,
+         solver="views/gcc-hipsycl/bin/Electrostatic2D3V",
+         conditionsFileName="two_stream_conditions.xml",
+         meshFileName="two_stream_mesh.xml",
+         **args):
+
+    update_environment(args)
+    with_config(config)
+    execute(put_configs, config)
+
+    env.neso_solver = solver
+
+    #This we presumably change somehow so that it gets changed throughout
+    #the SWEEP dir?
+    env.neso_conditions_file = (find_config_file_path(config)+
+                                "/" + conditionsFileName)
+    #All of these should be in a config file somewhere
+    env.neso_mesh_file = find_config_file_path(config) + "/" + meshFileName
```

Comment:
Ideally when constructing paths, we should either use `pathlib.Path` objects which overload `/` operator to join path components, or use `os.path.join`, as this make sure the paths are robust to operating system differences in path separators etc.

So for example if we add a `from pathlib import Path` in the import section, here would do something like

```Python
    env.neso.mesh_file = Path(find_config_file_path(config)) / meshFileName
```